Scripts/BaseActor.gd

```diff
@@ -0,0 +1,8 @@
+extends "res://Scripts/Actor.gd"
+
+
+func interact(from_direction):
+	if GameData.object_picked_up:
+		target_position(from_direction)
+	else:
+		print("You lack that power")
```

Comment:
Oh, I thought we'd already added this - did we move it here from somewhere else?